docs/domains/Domain.json

```diff
@@ -354,6 +359,11 @@
           "type": "number",
           "minimum": 0
         },
+        "maxClusterConcurrentShutdown": {
+          "description": "The default maximum WebLogic Server instances that a cluster will shutdown in parallel when it is being partially shutdown by lowering its replica count. You can override this default on a per cluster basis by setting the cluster\u0027s `maxConcurrentShutdown` attribute. A value of 0 means there is no limit. Defaults to 1",
```

Comment:
The default maximum WebLogic Server instances -> The default maximum number of WebLogic Server instances
shutdown -> shut down 
Defaults to 1", -> Defaults to 1.",